deps/openssl/openssl/crypto/bio/bss_fd.c

```diff
@@ -90,7 +90,7 @@ BIO_METHOD *BIO_s_fd(void)
  * Trouble is that unlike Unix file descriptors [which are indexes
  * in kernel-side per-process table], corresponding descriptors on
  * platforms which require "UPLINK" interface seem to be indexes
- * in a user-land, non-global table. Well, in fact they are indexes
+ * in a userland, non-global table. Well, in fact they are indexes
```

Comment:
Probably don't want to change stuff in `deps` because that's source code the project gets from elsewhere rather than maintains itself.